README.md

```diff
@@ -36,6 +36,21 @@ brew update
 brew upgrade unused
 ```
 
+### Nix
+
+There is a [Nix] expression [available in nixpkgs].
+
+There are many ways to run `unused` with Nix, but the simplest is:
+
+```sh
+nix run -f https://github.com/NixOS/nixpkgs/archive/nixpkgs-unstable.zip unused -c unused --help
```

Comment:
This should probably be nix-shell.

Reply:
Ah. I don't know how to do the same with nix-shell, could you please translate?

Reply:
```
nix-shell -p unused --run unused --help
```

If you just do ``nix-shell -p unused`` it opens a bash shell with unused in your path.

Reply:
That brings up the man page for nix-shell, and 
```
nix-shell -p unused --run 'unused --help'
```
gives me 
```
error: undefined variable 'unused' at (string):1:94
(use '--show-trace' to show detailed location information)
```

Is there a nix-shell one-liner to launch `unused` without manipulating channels? Note we are aiming for the simplest method here — everyone is going to have their own preferred way to use it.

Reply:
Forgot the quotes.

Works for me:
```
nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/nixpkgs-unstable.tar.gz -p unused --run 'unused --help'
```

Reply:
Thanks! I'll make another PR I guess.